### DISCUSSION TITLE
Is FindMultiModalNeighbors function needed after subsetting

### DISCUSSION THREAD

**Discussion by sylestiel** [stuart-lab/signac#1319] [General]
Hi,
 
With respect to a Multiome dataset, once I subset the data to retain certain clusters do I need to re-run the FindMultiModalNeighbors() function again or can I proceed without it?

Thanks!

**Comment by timoast** [comment]
I would recommend that you re-compute dimension reductions for each assay in the subset data (including re-running FindVariableFeatures and ScaleData [RNA] and RunTFIDF [ATAC]), and then run FindMultiModalNeighbors()
